frontend/src/pages/Home.jsx

```diff
@@ -1,14 +1,31 @@
-import Poster from "../components/Poster";
+import axios from "axios";
+import { useState, useEffect } from "react";
 import Answer from "../components/Answer";
+import Poster from "../components/Poster";
+
+const test = Math.floor(Math.random() * 150000) + 12;
 
 export default function Home() {
+  const [movie, setMovie] = useState([]);
+
+  function film() {
```

Comment:
the function is well written, but name it with an action: getFilm ? fetchFilm ? ...